src/Php81/bootstrap.php

```diff
@@ -18,3 +18,7 @@
 if (!function_exists('array_is_list')) {
     function array_is_list(array $array): bool { return p\Php81::array_is_list($array); }
 }
+
+if (!function_exists('enum_exists')) {
+    function enum_exists(string $enum, bool $autoload = true): bool { return $autoload && class_exists($enum, $autoload) && false; }
```

Comment:
you could omit the `$autoload` argument of `class_exist` as `false` will never reach that point (or remove the `$autoload && ` before it)

Reply:
true, updated.